apache/superset
# feat: Add label and tooltip for the color schemes control

### SUMMARY
This PR adds the label name in the color schemes control to make it easier for the users to find their color schemes.
It also adds a tooltip when the color scheme label or the color list gets truncated.

### BEFORE

![Screenshot 2022-08-10 at 22 05 46](https://user-images.githubusercontent.com/60598000/183996721-d2558c5b-2089-48e3-a69a-3c25462bc5cd.png)

### AFTER

![Screenshot 2022-08-10 at 22 02 09](https://user-images.githubusercontent.com/60598000/183996122-99bb7c6d-d8fe-480a-896a-3cb98aadac5e.png)


### TESTING INSTRUCTIONS
1. Open Explore
2. Scroll the available color schemes
3. If the label and/or the colors are truncated a tooltip should appear
4. If the label and the colors are fully visible, the tooltip should not appear

### ADDITIONAL INFORMATION
